doc/content/verification_and_validation/ver-1gc.md

```diff
@@ -53,12 +53,18 @@ The time evolution of the species concentrations from the analytical solution is
 
 ## Results and comparison against analytical solution
 
-The comparison of TMAP8 results against the analytical solution is shown in [ver-1gc_comparison_diff_conc]. The match between TMAP8's predictions and the analytical solution is satisfactory, with root mean square percentage errors (RMSPE) of RMSPE = 0.54 % for species $A$, RMSPE = 0.36 % for species $B$, and RMSPE = 0.04 % for species $C$. The larger values of RMSPE for species $A$ and $B$ are due to the small average values of these concentrations over time, and do not reflect on the accuracy of the TMAP8 solve itself.
+The comparison of TMAP8 results against the analytical solution is shown in [ver-1gc_comparison_diff_conc]. The match between TMAP8's predictions and the analytical solution is satisfactory, with root mean square percentage errors (RMSPE) of RMSPE = 4.78 % for species $A$, RMSPE = 3.20 % for species $B$, and RMSPE = 0.38 % for species $C$. The larger values of RMSPE for species $A$ and $B$ are due to the small average values of these concentrations over time, and do not reflect on the accuracy of the TMAP8 solve itself.
```

Comment:
I'm surprised the RMSPE is increasing so much (x10) since the documentation was first created!
Was it wrong in the first place?

Reply:
There is nothing wrong in the python script. It just output the new RMSPE values different from the values on documentation. Maybe the gold files are updated in previous commits?